packages/pebble-web/src/components/DropDown.tsx

```diff
@@ -47,9 +47,11 @@ class DropDown extends React.PureComponent<DropdownProps, DropdownState> {
       isSelected,
       disabled,
       labelClassName,
+      controlled,
+      isOpen,
```

Comment:
dont pluck `isOpen` from props. this will silently replace `isOpen` that was previously plucked from state.